apis/core/v1alpha1/backupconfiguration_types.go

```diff
@@ -291,31 +295,268 @@ type RepositoryInfo struct {
 
 // VerificationStrategy specifies a strategy to verify the backed up data.
 type VerificationStrategy struct {
-	// Name indicate the name of this strategy
+	// Name indicates the name of this strategy.
 	Name string `json:"name,omitempty"`
 
-	// Repository specifies the name of the repository which data will be verified
-	Repository string `json:"repository,omitempty"`
+	// RestoreOption specifies the restore target, addonInfo and manifestOption for backup verification
+	// +optional
+	RestoreOption *RestoreOption `json:"restoreOption,omitempty"`
 
-	// Verifier refers to the BackupVerification CR that defines how to verify this particular data
-	Verifier *kmapi.TypedObjectReference `json:"verifier,omitempty"`
+	// VerifySchedule specifies the schedule of backup verification in Cron format, see https://en.wikipedia.org/wiki/Cron.
+	VerifySchedule string `json:"verifySchedule,omitempty"`
 
-	// Params specifies the parameters that will be used by the verifier
-	// +kubebuilder:pruning:PreserveUnknownFields
+	// KeepAlive specifies the duration of keeping the instances created for backup verification.
 	// +optional
-	Params *runtime.RawExtension `json:"params,omitempty"`
+	// KeepAlive *metav1.Time `json:"keepAlive,omitempty"`
+
+	// Function specifies the name of a Function CR that defines a container definition
+	// which will execute the verification logic for a particular application.
+	Function string `json:"function,omitempty"`
+
+	// Volumes indicates the list of volumes that should be mounted on the verification job.
+	Volumes []ofst.Volume `json:"volumes,omitempty"`
 
-	// VerifyEvery specifies the frequency of backup verification
-	// +kubebuilder:validation:Minimum=1
-	VerifyEvery int32 `json:"verifyEvery,omitempty"`
+	// VolumeMounts specifies the mount for the volumes specified in `Volumes` section
+	VolumeMounts []core.VolumeMount `json:"volumeMounts,omitempty"`
 
 	// OnFailure specifies what to do if the verification fail.
 	// +optional
-	OnFailure FailurePolicy `json:"onFailure,omitempty"`
+	// OnFailure FailurePolicy `json:"onFailure,omitempty"`
```

Comment:
What about the `FailurePolicy` field?

Reply:
We still don't have any plan regarding `FailurePolicy`. You can find that we also removed this field from backup and restore.